test/e2e_source_map_test.ts

```diff
@@ -22,9 +22,6 @@ import {createOutputRetainingCompilerHost, createSourceReplacingCompilerHost, to
 
 import * as testSupport from './test_support';
 
-describe('source maps with TsickleCompilerHost', () => {
```

Comment:
Maybe keep this for now, and remove the tests for `TsickleCompilerHost` when the class itself is removed?